dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -1037,6 +1078,189 @@ protected void setSharedLib(Library lib) {
+                }
+            else 
+                formatter = defaultFormatter;
+        } catch (Exception e) {
+            formatter = defaultFormatter;
+            parseExceptions.put(ORACLELOG_FORMAT, e);
+        }
+        
+        holder = System.getProperty(ORACLELOG_TRACELEVEL);
+        try {
+            if (holder != null && !holder.equals(""))
+                traceLevel = Level.parse(holder);
+            else 
+                traceLevel = defaultLevel;
+        } catch (IllegalArgumentException e) {
+            traceLevel = defaultLevel;
+            parseExceptions.put(ORACLELOG_TRACELEVEL, e);
+        }
+        
+        if (tc.isWarningEnabled() && !parseExceptions.isEmpty()) {
+            for(Map.Entry<String, Exception> entry : parseExceptions.entrySet()) {
+                Tr.warning(tc, "ORACLE_TRACE_PARSE_WARNING", entry.getKey(), entry.getValue().getLocalizedMessage());
+            }
+        }
+        
+        /* "%g" the generation number to distinguish rotated logs 
+         * "%u" a unique number to resolve conflicts 
+         * If user provided oracle.log, then log file generated will be oracle.0.0.log
+         * If user provided oracle, then log file generated will be oracle.0.0.log
+         */
+        int directoryEnds = fileName.contains("/") ? fileName.lastIndexOf("/") : 0;
+        String originalFileName = fileName;
+        fileName = fileName.substring(0, directoryEnds) + (
+                        fileName.contains(".") ? 
+                                        fileName.substring(directoryEnds).replaceFirst("\\.", ".%g.%u.") : 
+                                        fileName.substring(directoryEnds).concat(".%g.%u.log"));
+        
+        if (tc.isDebugEnabled()) {
+            Tr.debug(tc, "ORACLELOG_FILENAME is:  " + originalFileName); 
+            Tr.debug(tc, "ORACLELOG_PACKAGENAME is: " + packageName); 
+            Tr.debug(tc, "ORACLELOG_FILE_SIZE_LIMIT is: " + fileSizeLimit); 
+            Tr.debug(tc, "ORACLELOG_FILE_COUNT is: " + fileCountLimit); 
+            Tr.debug(tc, "ORACLELOG_FORMAT is: " + formatter.getClass()); 
+            Tr.debug(tc, "ORACLELOG_TRACELEVEL is: " + traceLevel.getName()); 
+            Tr.debug(tc, "File name provided to java.util.logging: " + fileName);
+        }
+        
+        //Get and modify logger(s)
+        Logger logger = Logger.getLogger(packageName);
+        Handler handler;
+
+        try {
+            handler = new FileHandler(fileName, fileSizeLimit, fileCountLimit);
+            handler.setFormatter(formatter);
+            handler.setLevel(Level.ALL); //Logger should determine what gets logged not the handler
+             
+            parentLogger.setLevel(traceLevel); //Parent logger doesn't need to log anymore than the child (if one exists)
+            parentLogger.setUseParentHandlers(false); //Make sure this logger does not use WAS Logging
```

Comment:
This is already false because at the start of this method, you do an immediate return if UseParentHandlers isn't false

Reply:
Incorrect, at the start of the method I check if `UseParentHandlers` is set to false, then return. 
Therefore, at this point, `UseParentHandlers` is true, and needs to be set to false.

Reply:
ayup, you're right